lib/internal/Magento/Framework/Logger/Handler/System.php

```diff
@@ -3,10 +3,13 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
 
 namespace Magento\Framework\Logger\Handler;
 
+use Exception;
 use Magento\Framework\Filesystem\DriverInterface;
+use Magento\Framework\Logger\Handler\Exception as ExceptionHandler;
```

Comment:
I think it's better to name it HandlerException, as ExceptionHandler means that it will handle some exception(s).
```suggestion
use Magento\Framework\Logger\Handler\Exception as HandlerException;
```

Reply:
Hello @ihor-sviziev 
Thank you for your suggestion.
The property of this class has already been named `$exceptionHandler`. Why do we need to change the style of the alias?
Also, the style of alias naming like `xxxxxHandler` has already existed in Magento.

Reply:
@ihor-sviziev I think ExceptionHandler is the better name, as this class is a handler, not an exception

Reply:
@sivaschenko @andrewbess it definitely make sense. From the class name I just thought it’s opposite